spec/ParseUser.spec.js

```diff
@@ -1813,22 +1870,42 @@ describe('Parse.User testing', () => {
     };
     defaultConfiguration.auth.shortLivedAuth.setValidAccessToken('token');
     Parse.User._registerAuthenticationProvider(provider);
-    Parse.User._logInWith('shortLivedAuth', {})
-      .then(() => {
-        // Simulate a remotely expired token (like a short lived one)
-        // In this case, we want success as it was valid once.
-        // If the client needs an updated one, do lock the user out
-        defaultConfiguration.auth.shortLivedAuth.setValidAccessToken('otherToken');
-        return Parse.User._logInWith('shortLivedAuth', {});
-      })
-      .then(
-        () => {
-          done();
-        },
-        err => {
-          done.fail(err);
-        }
-      );
+    await Parse.User._logInWith('shortLivedAuth', {});
+    // Simulate a remotely expired token (like a short lived one)
```

Comment:
What's a "remotely expired token"?

Reply:
i do not introduced this comment but i believe that a "remotely expired token" is an expired token in a 3rd party database. So the 3rd party will return that the provided client token is expired and will not login the user.

Reply:
Why has this been resolved? We are both not sure what it means apparently? Needs rewording.

Reply:
This discussion is irrelevant for merging the PR. This comment was not created or changed by this PR. @Moumouls just moved from "then" to "await". Please look at the whole code change and you will see the comment was already there.

Reply:
Didn't notice this was just moved, yes, irrelevant for this PR

Reply:
+1 for @rhuanbarreto comment. This comment is OOS (out of scope). I think it's now time to merge this one.